style.css

```diff
@@ -1,2 +1,199 @@
-body { font-family: "Montserrat", serif; }
+body { 
+  background: #FFF;
```

Comment:
Codacy found an issue: [Expected "#FFF" to be "#fff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6044074)